vertx-web/src/test/java/io/vertx/ext/web/RouterTest.java

```diff
@@ -1735,7 +1735,7 @@ public void testLocaleWithCountry() throws Exception {
       rc.response().end();
     });
 
-    testRequest(HttpMethod.GET, "/foo", req -> req.putHeader("Accept-Language", "en-gb;q=0.8, en;q=0.7, da_DK;q=0.9"), 200, "OK", null);
```

Comment:
I may return it but that's not what [rfc4647#section-2.1](https://tools.ietf.org/html/rfc4647#section-2.1) states.
To quote it:

> language-range   = (1*8ALPHA *("-" 1*8alphanum)) / "*"

How do you want to follow? Accept underscore (`_`) as a separator in addition to hyphen (`-`) or stay like I did?